R/module_demand.R

```diff
@@ -80,11 +80,11 @@ demand_server <-  function(id, params, upload_event) {
           s_input <- numericInput(NS(id, s_name), NULL, suppressed, min = 0, step = 1)
 
           demand_observables[[u_name]] <<- observeEvent(input[[u_name]], {
-            params$demand[[input$service]]$underlying[[month_ix]] <- input[[u_name]]
+            demand$underlying[[month_ix]] <- input[[u_name]]
```

Comment:
This breaks test in test-module_demand "it updates params correctly": it's updating the copy of params$demand, not params$demand directly.